backend/moonstreamapi/routes/dashboards.py

```diff
@@ -427,10 +428,37 @@ async def get_dashboard_data_links_handler(
                     ExpiresIn=300,
                     HttpMethod="GET",
                 )
-                stats[subscription.id][timescale] = stats_presigned_url
+                stats[subscription.id][timescale] = {"url": stats_presigned_url}
             except Exception as err:
                 logger.warning(
                     f"Can't generate S3 presigned url in stats endpoint for Bucket:{MOONSTREAM_S3_SMARTCONTRACTS_ABI_BUCKET}, Key:{result_key} get error:{err}"
                 )
 
     return stats
+
+
+@router.post("/{dashboard_id}/stats_update", tags=["dashboards"])
+async def update_dashbord_data_handler(
+    request: Request, dashboard_id: str, updatestats: data.UpdateStats
```

Comment:
In future it will help to read code.

```python
dashboard_id: str = Path(...),
updatestats: data.UpdateStats = Body(...),
````

Reply:
add